system/jlib/jstats.cpp

```diff
@@ -1825,7 +1813,7 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
     }
     virtual StringBuffer & getFullScope(StringBuffer & str) const override
     {
-        if (parent)
+        if (parent && queryScopeType()!=SSTworkflow)
```

Comment:
better test would be parent->queryscopeType() != SSTglobal.  (Would work with compile stages as well.)